scipy/sparse/csr.py

```diff
@@ -100,6 +100,16 @@ class csr_matrix(_cs_matrix):
            [0, 0, 3],
            [4, 5, 6]])
 
+    Duplicate entries are summed together:
+
+    >>> row = np.array(0, 1, 2, 0)
```

Comment:
the numbers should be in square brackets: `np.array([0, 1, 2, 0])`

Reply:
Yep, sorry